topics/cdnjs/index.md

```diff
@@ -1,11 +1,11 @@
 ---
-created_by: Ryan Kirkman, Thomas Davis, Peter Dave, Matt Cowley
+created_by: Matt Cowley, Sven Sauleau, Tyler Caslin, Drew Freyling, Alex, Ryan Kirkman, Thomas Davis
 display_name: cdnjs
-github_url: https://github.com/cdnjs/cdnjs
+github_url: https://github.com/cdnjs/packages
```

Comment:
What is the intention of this change? cdnjs/cdnjs still seems like a valid repo to point to.

Reply:
I can probably provide some context here: cdnjs/cdnjs is the repo that contains the content hosted on the CDN, but is now locked down and robot-only. cdnjs/packages is the new repo that we encourage folks to contribute to, where all the hosted packages are defined.

I assume the intention with this change is to keep this file pointing at a contributor-friendly repo?

Reply:
I just checked and we support org level links here. This would be more appropriate in this instance since we aren't using this page to direct visitors on how to contribute to cdnjs, it's more of a place to give context on what a topic is.
```suggestion
github_url: https://github.com/cdnjs
```